pyproject.toml

```
@@ -96,11 +96,11 @@ dev = [
     # Also update [build-system] -> requires
     'meson-python>=0.13',
     'wheel',
-    'setuptools>=67',
+    'setuptools==69.5.1',
```

Perhaps more flexible, but still ensuring it's locked in to the 69 major.
```suggestion
    'setuptools~=69.5',
```